cli/azd/internal/scaffold/spec.go

```diff
@@ -69,3 +69,43 @@ func containerAppExistsParameter(serviceName string) Parameter {
 		Type: "bool",
 	}
 }
+
+type serviceDef struct {
+	Settings []serviceDefSettings `json:"settings"`
+}
+
+type serviceDefSettings struct {
+	Name         string `json:"name"`
+	Value        string `json:"value"`
+	Secret       bool   `json:"secret,omitempty"`
+	SecretRef    string `json:"secretRef,omitempty"`
+	CommentName  string `json:"_comment_name,omitempty"`
+	CommentValue string `json:"_comment_value,omitempty"`
+}
+
+func serviceDefPlaceholder(serviceName string) Parameter {
+	return Parameter{
+		Name: BicepName(serviceName) + "Definition",
+		Value: serviceDef{
+			Settings: []serviceDefSettings{
+				{
+					Name:        "",
+					Value:       "${VAR}",
+					CommentName: "The name of the environment variable when running in Azure. If empty, ignored.",
```

Comment:
Should these long strings be constants?

Reply:
I think it makes as a constant if it needs to be referenced in multiple places that isn't local to the functioning code. Not a strong need right now